### DISCUSSION TITLE
TEST BETA

### DISCUSSION THREAD

**Discussion by SirHautDeFraise** [dussartcorp/Gsb#15] [General]
Oh en faite plein de maj de github (on peut le mettre en dark mode et tout)

- [x] Etre incroyable

- [x] Faire du café

**Comment by Neguro** [comment]
test numéro deux yeaaah boiiiiiiiiii

**Reply by SirHautDeFraise** [reply]
J'adore UwU 

- [ ] Donner de l'argent à Thibault

**Reply by Neguro** [reply]
jamais tu me doit 0.10 cts

**Comment by DUSSARTLuke** [comment]
Bonjour chers collaborateurs, veuillez prendre note de ma participation 😉
